app/components/datafruits-chat.js

```diff
@@ -30,8 +30,13 @@ export default class DatafruitsChat extends Component {
   joinedUsers;
 
   @computed('isJoiningChat', 'nick.length')
+  @action
+  agreeChanged() {
+    // input handler requires a function to exist for this to work correctly
+    // ¯\_(~_~)_/¯
+  }
```

Comment:
It works for me if I remove this and add `agreeToCoC` to the list of computed dependencies above.